bindings/java/build-unix.xml

```diff
@@ -27,6 +27,42 @@
 		</java>
 	</target>
 
+	<condition property="online">
+		<http url="https://www.google.com/"/>
+	</condition>
+
+	<target name="set-library-path"  description="sets the path of the libraries" depends="set-library-path-online,set-library-path-offline">
+
+	</target>
+
+	<target name="set-library-path-online" description="set this library path when the user is online" if="online">
+		<path id="libraries">
+		<fileset dir="${lib}">
```

Comment:
Can you look into making the indentation consistent for both this and the offline target?